src/utils/selectors.ts

```diff
@@ -10,3 +10,11 @@ export const getAnnotation = (
 // labels
 export const getLabel = (entity: K8sResourceCommon, label: string, defaultValue?: string): string =>
   entity?.metadata?.labels?.[label] ?? defaultValue;
+
+// conditions
+export const getConditionReason = (condition) => condition?.reason;
```

Comment:
add types to function parameters please

Reply:
we have the generic interface:
https://kubevirt-ui.github.io/kubevirt-api/interfaces/kubernetes_models.IoK8sApiCoreV1ComponentCondition.html

but we may want something else once we move to the components repo

Reply:
@metalice is this PR [1] will also expose `K8sResourceKind`? if so, I can wait for it and use this as a type for my entity argument
[1] https://github.com/openshift/console/pull/11095